.github/workflows/mongodb-version-check.yml

```diff
@@ -0,0 +1,82 @@
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '0 0 1 */2 *' # run check at midnight, at the first day of the month, every two months
+
+defaults:
+  run:
+    shell: bash
+jobs:
+  check-mongodb-versions:
+    name: Check MongoDB chart versions
+    runs-on: ubuntu-20.04
+    outputs:
+      CREATE_ISSUE: ${{ steps.compare_mongodb_versions.outputs.CREATE_ISSUE }}
+      BITNAMI_VERSION: ${{ steps.get_bitnami_mongodb_version.outputs.MONGODB_VERSION }}
+      KEPTN_VERSION: ${{ steps.get_keptn_mongodb_version.outputs.MONGODB_VERSION }}
+    steps:
+    - name: Check out code.
+      uses: actions/checkout@v3
+
+    - name: "Get keptn mongodb version"
+      id: get_keptn_mongodb_version
+      run: |
+        MONGODB_VERSION=$(cd ./installer/manifests/keptn/ && helm dependency list | grep -e "mongodb" | awk '{print $2}')
+        echo "::set-output name=MONGODB_VERSION::$MONGODB_VERSION"
+  
+    - name: "Get latest bitnami/mongodb version"
+      id: get_bitnami_mongodb_version
+      run: |
+        helm repo add bitnami https://charts.bitnami.com/bitnami
+        MONGODB_VERSION=$(helm search repo bitnami | grep -e "mongodb " | awk '{print $2}')
+        echo "::set-output name=MONGODB_VERSION::$MONGODB_VERSION"
+
+    - name: "Compare MongoDB Chart versions"
+      id: compare_mongodb_versions
+      env:
+        KEPTN_VERSION: ${{ steps.get_keptn_mongodb_version.outputs.MONGODB_VERSION }}
+        BITNAMI_VERSION: ${{ steps.get_bitnami_mongodb_version.outputs.MONGODB_VERSION }}
+      run: |
+        CREATE_ISSUE=false
+        if [[ "$KEPTN_VERSION" != "$BITNAMI_VERSION" ]]; then
+          CREATE_ISSUE=true
+        fi
+        echo "::set-output name=CREATE_ISSUE::$CREATE_ISSUE"
+
+  create_issue:
+    name: Create GitHub Issue
+    runs-on: ubuntu-20.04
+    needs: check-mongodb-versions
+    steps:
+    - name: Formulate bug issue
+      if: needs.check-mongodb-versions.outputs.CREATE_ISSUE == 'true'
+      id: formulate_bug_issue
+      env:
+        KEPTN_VERSION : ${{ needs.check-mongodb-versions.outputs.KEPTN_VERSION }}
+        BITNAMI_VERSION : ${{ needs.check-mongodb-versions.outputs.BITNAMI_VERSION }}
+      run: |
+        # create a markdown file that contains details about the error
+        echo "---" > mongodb-version-changed.md
+        echo "title: 'Bitnami/MongoDB version changed'" >> mongodb-version-changed.md
+        echo "labels: ['type:critical', 'area:core']" >> mongodb-version-changed.md
+        echo "---" >> mongodb-version-changed.md
+        echo "" >> mongodb-version-changed.md
+        echo "* Link to run: $GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID" >> mongodb-version-changed.md
+        if [[ $GITHUB_EVENT_NAME == 'schedule' ]]; then
+          echo "* Triggered by: Scheduled build" >> mongodb-version-changed.md
+        else
+          echo "* Triggered by: @$GITHUB_ACTOR" >> mongodb-version-changed.md
+        fi
```

Comment:
you don't allow any other triggers than schedule so I don't think you need this if here